src/index.js

```diff
@@ -0,0 +1,5 @@
+import './index.css';
+import { Application } from '../modules/application/application.js';
+/* eslint-disable */
+const application = new Application();
+/* eslint-enable */
```

Comment:
A linter is a tool that analyzes your source code to flag programming errors, bugs, stylistic errors, and suspicious constructs(source: [Wikipedia](https://en.wikipedia.org/wiki/Lint_(software))). Please do NOT disable them.